reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
+        """
+        Remove Gerrit's prefix and convert to JSON.
+        Returns:
+            Converted JSON content
+        Raises:
+            ValueError if the content is not in proper json format.
+        """
+        gerrit_json_prefix = ")]}\'\n"
+        content = response.content.strip()
+        try:
+            if response.encoding:
+                content = content.decode(response.encoding)
+            response.raise_for_status()
+            content_type = response.headers.get('content-type', '')
+            if content_type.split(';')[0] != 'application/json':
```

Comment:
Gerrit sends json response but has it's prefix along with it. 
Example:

> )]}'
[
  {
    "id": "gerrit~master~Ia4bec7eac8db700bbce00165c7f5b8687c62a554",

So if we call` response.json()` it throws` ValueError`. Because of this, we have to remove the prefix added and then convert the content back to json.